featuremanagement/_defaultfilters.py

```diff
@@ -82,14 +82,19 @@ def _is_targeted(context_id, rollout_percentage):
         if rollout_percentage == 100:
             return True
 
-        hashed_context_id = hashlib.sha256(context_id.encode()).hexdigest()
-        context_marker = abs(int(hashed_context_id, 16))
-        percentage = (context_marker / (2**256 - 1)) * 100
+        hashed_context_id = hashlib.sha256(context_id.encode()).digest()
 
+        byte_array = bytearray(hashed_context_id)
+        decimal_values = [int(byte) for byte in byte_array[:4]]
```

Comment:
This line doesn't do anything as far as I can tell. We can just pass `byte_array[:4]` into the next line instead.

Reply:
I think that section got added as debug steps. Removing.